code/modules/surgery/organs/organ.dm

```diff
@@ -229,7 +232,15 @@
 	status &= ~ORGAN_SPLINTED
 	status |= ORGAN_ROBOT
 
-/obj/item/organ/proc/remove(mob/living/user, special = 0)
+/*
+  * remove
+  *
+  * Removes the organ from the user properly.
+  * If the organ is vital, it will kill the user.
+  * The proc returns the organ removed (i.e. `src`) assuming it was removed successfully;
+* otherwise, or if the organ gets destroyed in the process, it returns null.
+*/
+/obj/item/organ/proc/remove(mob/living/user, special = FALSE, )
```

Comment:
Since special isn't always true or false (abductor surgery), you might want to revert this back to a numerical argument. Also, you might want to document what's going on with special and why it's numerical

```suggestion
/obj/item/organ/proc/remove(mob/living/user, special = 0)
```

Reply:
Also, you might want to add the SHOULD_CALL_PARENT spacemandmm hint to this function if they need to call the parent, at least for consistency.